src/common/BackBtn/BackBtn.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import * as UI from "./style";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowLeft } from "@fortawesome/free-solid-svg-icons";
+
+export interface BackLinkProps {
+  url: string;
+}
+
+export const BackBtn: React.FC<BackLinkProps> = (props: BackLinkProps) => {
```

Comment:
This can probably be renamed something different.
Or it might be more useful to just make this component the Link:
```
<Container>
    <BackBtn url={foo}/>
</Container>
```

Reply:
kinda confused what you mean here. Can you look at what I did so far and tell me what you want changed? Do you want the name to be more generic?